neurolib/models/multimodel/model.py

```diff
@@ -128,27 +126,22 @@ def run(
         chunkwise=False,
         chunksize=None,
         bold=False,
-        append=False,
-        append_outputs=None,
+        append_outputs=False,
         continue_run=False,
         noise_input=None,
     ):
         self._update_model_params()
 
-        # TODO: legacy argument support
-        if append_outputs is not None:
-            append = append_outputs
-
         # if a previous run is not to be continued clear the model's state
-        if continue_run is False:
+        if continue_run:
+            self.setInitialValuesToLastState()
+        else:
```

Comment:
We would like to allow `continue_run` even in the case if there hasn't been any previous run.